src/backend/expungeservice/models/charge_types/sex_crimes.py

```diff
@@ -0,0 +1,91 @@
+from dataclasses import dataclass
+
+from expungeservice.models.charge import Charge
+from expungeservice.models.expungement_result import TypeEligibility, EligibilityStatus
+
+
+@dataclass(eq=False)
+class SexCrimes(Charge):
+    type_name: str = "Sex Crimes"
```

Comment:
Our charges types are all singular, so this should be too (The posted issue was misleading), and that can include the class name and the new annotations in `person_felony.py`